ui/src/components/pages/Pipeline/index.tsx

```diff
@@ -47,24 +102,61 @@ export function Pipeline() {
     if (watermarkErr) notifyError(watermarkErr);
   }, [watermarkErr]);
 
+  const { setSidebarProps } = useContext<AppContextProps>(AppContext);
+  const handleError = useCallback(() => {
+    setSidebarProps({
+      type: SidebarType.ERRORS,
+      errorsProps: {
+        errors: true,
+      },
+      slide: false,
+    });
+  }, [setSidebarProps]);
+
   if (pipelineErr || buffersErr) {
-    return <div>Error</div>;
+    return (
+      <div
```

Comment:
In many places, we use MUI Box. In others (such as here), we use div. What's the decision criteria for when we use Box vs. div?

Reply:
Its better to use MUI Box than plain div. We will consciously start using Box.